packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -0,0 +1,261 @@
+---
+title: System Log events for rate limits
+category: rate limits
+---
+
+# System Log events for rate limits
+
+The Okta [System Log](/docs/reference/api/system-log/) records system events related to your organization to provide an audit trail that you can use to understand platform activity and to diagnose problems. There are a few standard System Log events that you may encounter when Okta is enforcing [rate limits](/docs/reference/rate-limits/) on specific operations.
+
+* `system.operation.rate_limit.violation`<br>
+This event type is sent once per rate limiting period when a request is rejected for exceeding a rate limit. For example, if the rate limit that was exceeded has a reset period of one minute, then one event of this type is emitted during that period for the applicable scope.
+
+* `system.operation.rate_limit.warning`<br>
+This event type is sent once per rate limiting period as a warning that some significant portion of your rate limit has already been used within a period. For example, you receive a warning that you have reached 60% of your daily limit for event delivery.
```

Comment:
"is sent once per rate limiting period" -> "may be sent once per rate limiting period"

"For example, you receive a warning" -> "For example, you may receive a warning"

Reply:
"60% of your daily limit for event delivery" -> "60% of your rate limit for an endpoint within a rate limiting period"